src/rules/accessibility/required-wrapper-accessibility.js

```diff
@@ -0,0 +1,78 @@
+HTMLHint.addRule({
+    id: 'required-wrapper-accessibility',
+    description: 'div with requiredWrapper binding should contain at least one validation message and one element that points to it',
+    init: function (parser, reporter) {
+        var inRequiredWrapper = false;
+        var self = this;
+        var unclosedDivsCounter = 0;
+        var messagesInRequiredWrapper = [];
+        var usedMessagesInRequiredWrapper = [];
+
+        var isDivRequiredWraper = function (event) {
+            return HTMLHint.utils.getBindingValue(event, "requiredWrapper");
+        };
+
+        var isValidationMessage = function (event) {
+            return (event.tagName === "span" && HTMLHint.utils.isClassExsits(event.attrs, "validationMessage"));
+        };
+
+     
+
+        var findValidationMessageInDescription = function (description) {
+            var descriptionsIds = HTMLHint.utils.removeBoundaryQuotes(description).split(' ');
+            return descriptionsIds.find(function (descriptionId) {
+                return descriptionId.startsWith(HTMLHint.resources.validationMessagesAccessibility.messageIdPrefix);
+            });
+        };
+
+        var atleastOneBoundMessageInRequiredWrapper = function () {
+            if (messagesInRequiredWrapper.length === 0 || usedMessagesInRequiredWrapper.length === 0) {
+                return false;
+            }
+            return usedMessagesInRequiredWrapper.find(item => messagesInRequiredWrapper.indexOf(item) !== -1);
```

Comment:
how? I need to loop through both arrays simultaneously, and includes doesn't handle a callback function